test/python/circuit/library/test_mcmt.py

```diff
@@ -267,6 +268,19 @@ def test_vchain_open_control(self):
             )
         )
 
+    def test_mcmt_x_gate(self):
+        """Test MCMT gate uses `synth_mcmt_x` for X gate synthesis."""
+        for num_ctrl, num_targ in product(range(1, 6), range(2, 6)):
```

Comment:
It would be better to pull the `num_ctrl` and `num_targ` argument outside -- that way we'll see exactly which combination fails. See e.g. https://github.com/Qiskit/qiskit/blob/477cff7b6e550ece7188015e5761a22e04f4f796/test/python/circuit/test_controlled_gate.py#L563 for an example on how to do that 🙂

Reply:
Done